packages/flutter/test/material/dialog_theme_test.dart

```diff
@@ -12,27 +12,39 @@ import 'package:flutter/material.dart';
 import 'package:flutter/rendering.dart';
 import 'package:flutter_test/flutter_test.dart';
 
-MaterialApp _appWithDialog(WidgetTester tester, Widget dialog, { ThemeData? theme }) {
+MaterialApp _appWithDialog(WidgetTester tester, Widget dialog, { ThemeData? theme, DialogThemeData? dialogTheme }) {
```

Comment:
```suggestion
MaterialApp _appWithDialog(
  WidgetTester tester,
  Widget dialog, {
  ThemeData? theme,
  DialogThemeData? dialogTheme,
}) {
```

This line got bit lengthy.

Reply:
Updated!